cob_default_robot_config/cob4-2/behaviour.yaml

```diff
@@ -0,0 +1 @@
+service_ns: /behaviour
```

Comment:
do we really need this parameter? isn't behaviour the default? what other value would be used? in which cases?